MELA/makefile

```diff
@@ -43,7 +44,7 @@ ROOTINC = $(ROOFITINC)
 
 CC = g++
 CPPINC = -I$(MELADIR)/interface $(ROOTINC)
-CPPLOAD = -L$(MELALIBDIR) -lmcfm_709 -ljhugenmela -lcollier $(LIBS)
+CPPLOAD = -L$(MELALIBDIR) -lmcfm_709 -ljhugenmela -lcollier -L$(MADGRAPHLIBDIR) -lggF_SMEFTsim $(LIBS)
```

Comment:
Currently only ggF is supported - that is the test case for the future - but I can always rename the library.

Reply:
While it is certainly true that not everybody needs this functionality, the same could be said for MCFM. There are a lot of people who practically never use the MCFM matrix elements, yet are still required to compile MCFM. Given that we do not have a flag for MCFM, I don't see why we should have a flag for Madgraph.

Reply:
Yes, if we are sure that this will be a single library, we should keep its name generic and leave it to the documentation to describe that only ggH couplings are supported.

Reply:
> Yes, if we are sure that this will be a single library, we should keep its name generic and leave it to the documentation to describe that only ggH couplings are supported.

Actually I am unsure if this will be a single library. If we add a different process, say VBS for example, I'm not sure if that would be encompassed in the ggF library or a different one. I guess for now we can assume all processes can be subsumed into one with the renaming.

Reply:
renamed the library to `libSMEFTsim.so`. Any future libraries should attempt to be merged into this one through diagram filters or utilization of .o files.